bn-apps/memberships-management/membership-service-contracts-and-states/src/main/kotlin/com/r3/businessnetworks/membership/states/Membership.kt

```diff
@@ -34,6 +34,7 @@ open class MembershipContract : Contract {
     open class Commands : CommandData, TypeOnlyCommandData() {
         class Request : Commands()
         class Amend : Commands()
+        class SelfIssue : Commands()
```

Comment:
Is a separate command really required? Can we get away with reusing Request / Activate? Smart contract would need a bit of tweaking to allow bno and member to have the same identity though. Then BNO can have a SelfIssueMembershipFlow that is effectively a combination of Request + Activate.